WordPress/Classes/Services/PlanService.swift

```diff
@@ -31,92 +37,87 @@ open class PlanService: LocalCoreDataService {
         let remote = PlanServiceRemote(wordPressComRestApi: api)
         remote.getWpcomPlans({ plans in
 
-            self.mergeRemoteWpcomPlans(plans.plans, remoteGroups: plans.groups, remoteFeatures: plans.features, onComplete: {
-                success()
-            })
+            self.mergeRemoteWpcomPlans(plans.plans, remoteGroups: plans.groups, remoteFeatures: plans.features, onComplete: success)
```

Comment:
Nice 😄 No need to wrap a `() -> Void` in another `() -> Void` that only calls the first one.